examples/with-movement/src/index.ts

```diff
@@ -0,0 +1,146 @@
+import * as dotenv from "dotenv";
+import * as path from "path";
+import { AptosClient, TxnBuilderTypes, BCS, TransactionBuilder } from "aptos";
+import prompts from "prompts";
+import { Turnkey } from "@turnkey/sdk-server";
+import { bytesToHex } from "@noble/hashes/utils";
+
+dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
+
+async function main() {
+  const organizationId = process.env.ORGANIZATION_ID!;
+  const turnkeyClient = new Turnkey({
+    apiBaseUrl: process.env.BASE_URL!,
+    apiPublicKey: process.env.API_PUBLIC_KEY!,
+    apiPrivateKey: process.env.API_PRIVATE_KEY!,
+    defaultOrganizationId: organizationId,
+  });
+
+  const client = new AptosClient(
+    "https://testnet.bardock.movementnetwork.xyz/v1",
+  );
+  const movementAddress = process.env.MOVEMENT_ADDRESS!;
+  const movementPublicKeyHex = process.env.MOVEMENT_PUBLIC_KEY!;
+
+  if (!movementAddress || !movementPublicKeyHex) {
+    throw new Error(
```

Comment:
If it's not too much work, we can add a `createNewWallet` script that creates both of these if not provided

https://github.com/tkhq/sdk/blob/main/examples/with-viem/src/createNewWallet.ts